build-script.py

```diff
@@ -315,28 +349,36 @@ def build(self, product_name, module_group_path=""):
 
 
 def verify_generated_files(gyb_exec, verbose):
```

Comment:
I tried to run this locally to see if it worked, but this or `main` worked for me 🤷‍♂️